### ISSUE TITLE
Migrate away from badgekit

### ISSUE THREAD

**Issue by Wardormeur** [CoderDojo/community-platform#1092]
Badgekit is dead and burried, but now even Persona is giving up.
Instead of trying to maintain an abandoned soft by ourselves, migrate to badgr which seems still active.
TODOS: 
- [ ] map API integration (***)
- [ ] adapt josmas' image for production (*)
- [ ] migrate data (**)

Some references : 
https://github.com/concentricsky/badgr-server for production
https://github.com/josmas/local-badgr-server for
https://github.com/mozillascience/PaperBadger/issues/159

**Comment by Wardormeur** [comment]
see also https://github.com/CoderDojo/community-platform/issues/1055

**Comment by josmas** [comment]
hey @Wardormeur I thought you were only using the baking API; are you also using badgekit server and the client?

**Comment by Wardormeur** [comment]
I guess so :)
OpenBadgekit is what you refers as a client & Badgekit API to the baking one ?

**Comment by josmas** [comment]
Nope, the baking tool is (IIRC) a tool to bake information into a png. A **_badge_** as such is a png with some metadata (data like the person it was awarded to, who awarded it, an evidence url, and so forth). The [baking API](https://github.com/mozilla/openbadges-backpack/wiki/Badge-Baking) just does that. Unless things have changed in the last number of months, which is quite possible, Zen only used to use baking (but I might be wrong).

Hosting (and awarding) badges is part of what we do in [PaperBadger](https://github.com/mozillascience/PaperBadger) and for that we need a full badges server ([badgekit-api](https://github.com/mozilla/badgekit-api)) and a client([badgekit-api-client](https://github.com/mozilla/badgekit-api-client)). As you mention in this thread, they are not maintained, so we are migrating. So my question is, do you use _baking_ and _persona_ to push badges to a backpack, or you host and award the badges on your own infrastructure?

The openbadges spec is huge, and the terminology can be rather tricky and confusing.

**Comment by Wardormeur** [comment]
Ah, thanks for the details, lots of docs i definitiley need to read (just found the "Assertions definition" links from backpack, which are insightful :))
So no, we don't have backpack (even though it could be reconsidered considering the length of badging specs), Zen host the badges inside each user profile and they are awarded "from the inside" of Zen itself.
The definition of the badges and their requirements are, though, hosted by the badgekit API and the front-end our team use to manage those badges (openbadgekit) is based upon persona.

**Comment by josmas** [comment]
I see, so you are running badgekit-api as part of your infra; then yeah, a migration is in order! :)
Planning on doing 24pullrequests again this year, so if you guys haven't got around to working on this by then, I'll be happy to lend a hand!

**Comment by Wardormeur** [comment]
How about a hackhaton (& beers !) ? :)

**Comment by josmas** [comment]
That'd be cool too! :)

There's also http://24pullrequests.com/events

**Comment by Wardormeur** [comment]
updated to add format description of badges
